.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: Feature
+assignees: ''
+---
+
+#### Problem description
```

Comment:
I have used a level 4 heading here, because the same one is used in the current issue template. However, this might lead to a confusing structure, if the feature needs more explanation and more sub headings. I think I would go here for a level 1 heading, and I would do the same for the bug report. WDYT?

The only reason speaking against this is that it might take more space, but I think that is not a big issue. Putting some headlines here for a comparison:

# Heading 1
## Heading 2
### Heading 3
#### Heading 4
##### Heading 5
###### Heading 6

Personally I prefer clear structure over some visual aspects.